src/components/cards/CreatePool/PoolFees.vue

```diff
@@ -82,12 +82,15 @@ const feeOptions = FIXED_FEE_OPTIONS.map(option => {
  * FUNCTIONS
  */
 function onFixedInput(val: string): void {
+  fee.value = '';
   initialFee.value = '0';
   initialFee.value = val;
   isCustomFee.value = false;
 }
 
 function onCustomInput(val: string): void {
+  if (!val) return;
```

Comment:
if we check empty string here it's better to check empty string
`if (val === '')`
otherwise, arg type should be changed to `val?: string`

Reply:
Good point, changed